Logger.go

```diff
@@ -122,3 +117,26 @@ func getFrameInfo() string {
 
     return message
 }
+
+// parseArgs checks if the args have a value and replaces the value with nil (string)
+func parseArgs(args ...interface{}) []interface{} {
+    for key, value := range args {
+        reflectValue := reflect.ValueOf(value)
+
+        if isPointer(reflectValue) && reflectValue.IsNil() {
+            switch value.(type) {
+            case *string:
+              args[key] = "nil"
+            default:
+              args[key] = 0
```

Comment:
Waarom "0" en niet altijd "nil"? Ik denk dat het verwarrend kan zijn als je gaat loggen en je 0 te zien krijgt ipv nil. Ook met het loggen van een struct pointer krijg je nu 0 terwijl het geen integer is